src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
+  const headers = table.querySelectorAll('th');
+  const rows = Array.from(table.querySelectorAll('tbody tr'));
+
+  const sortRows = (index) => {
+    const isNumeric = index === 2 || index === 3;
```

Comment:
The condition for numeric sorting is hardcoded for columns with index 2 and 3. Ensure that these indices match the actual numeric columns in your table. If the table structure changes, this condition may need to be updated.